packages/dataviews/README.md

```diff
@@ -497,6 +480,383 @@ Parameters:
 
 Returns a boolean indicating if the item is valid (true) or not (false).
 
+## Fields API
+
+### `id`
+
+The unique identifier of the field.
+
+- Type: `string`.
+- Required.
+
+Example:
+
+```js
+{ id: 'field_id' }
+```
+
+### `type`
+
+Field type. One of `text`, `integer`, `datetime`.
```

Comment:
The list of supported types is interesting. There are differences between block attributes or custom post type meta fields. Where can I learn more about plans related to that?